state-chain/chains/src/btc.rs

```diff
@@ -15,7 +15,7 @@ use bech32::{self, u5, FromBase32, ToBase32, Variant};
 pub use cf_primitives::chains::Bitcoin;
 use cf_primitives::{
 	chains::assets, NetworkEnvironment, DEFAULT_FEE_SATS_PER_KILO_BYTE, INPUT_UTXO_SIZE_IN_BYTES,
-	MINIMUM_BTC_TX_SIZE_IN_BYTES, OUTPUT_UTXO_SIZE_IN_BYTES,
+	MINIMUM_BTC_TX_SIZE_IN_BYTES, MINIMUM_FEE_SATS_PER_KILO_BYTE, OUTPUT_UTXO_SIZE_IN_BYTES,
```

Comment:
ik this was this way before in some places, but kilobyte is one word, so it should be `MINIMUM_FEE_SATS_PER_KILOBYTE`,